solutions/only_ints.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for verifying integer inputs.
+
+Module contents:
+    - only_ints: A function to check if two parameters are both integers.
+
+Created on 12 29 2024
+@author: Mohammad
+"""
+
+
+def only_ints(param1, param2):
+    """
+    Using this function, you will be able to take two parameters, and it will return True
+    if both parameters are integers, and otherwise it is false
+
+    Parameters:
+    param1 (int): The first parameter to check, expected to be an integer.
+    param2 (int): The second parameter to check, expected to be an integer.
+
+    Returns:
+    bool: True if both parameters are integers, False otherwise.
+
+    Raises:
+    TypeError: If any of the parameters is not an integer.
```

Comment:
I dont think this sentence is correct. If any of the parameter are not integer it turns false not typeerror.

Reply:
How about if the change the if statement to something like this 

if not isinstance(param1, int) or not isinstance(param2, int):
        raise TypeError("Both parameters must be integers.")  # Raise TypeError

    return True
    I just tests it and if any of the parma is not integers , it will be a Raise TypeError with a meassge that says " TypeError: Both parameters must be integers."

Reply:
Hi Meklit,

I wanted to update you on the testing for the only_ints function again, While working on the test cases, I noticed that the Raises clause in the docstring was causing some confusion. Specifically, the function does not raise a TypeError as stated—it simply returns False for non-integer inputs.

After reviewing the implementation and testing, I removed the Raises section from the docstring to ensure it accurately reflects the function's behavior. This should help avoid further confusion moving forward.

If there is no review from you guys so I will finish it and push back with the updated version 

Thank you